include/cadmium/basic_model/accumulator.hpp

```diff
@@ -129,9 +129,16 @@ namespace cadmium {
                 return (std::get<on_reset>(state) ? TIME{} : std::numeric_limits<TIME>::infinity());
             }
         };
-
     }
 }
 
+std::ostringstream& operator<<(std::ostringstream& os, const std::tuple<int, bool>& s) {
```

Comment:
you can use ./include/cadmium/logger/tuple_to_ostream.hpp
If need to print a tuple.

Reply:
Done!

I included the suggested file in the accumulator.hpp file so the state can be printed.
The dynamic simulator requires the model states to be printable.